functional-tests/devscripts/run_tests.sh

```diff
@@ -54,6 +54,10 @@ else
 	TEST_SUITE="simpleTestSuite.xml"
 fi
 
+#This format allows us to see username even if it is placed in Jenkins credential store. 
+USERNAME_TO_PRINT="${USERNAME:0:3} ${USERNAME:3:${#USERNAME}}"
+echo "User name printed in format: 3 first letters, space, the rest of letters.    $USERNAME_TO_PRINT"
```

Comment:
Shouldn't this be 4 letters?

Reply:
I don't know why but if you want, I'm ok with 4 letters too. :) Should I change it?

Reply:
Ah no, I was just confused, never mind.

`User name printed in format: 3 first letters, space, the rest of letters. osio test2.` would be `osi otest2` in this case, right?

Reply:
Heh, yes, this is my typo. Sorry for confusion.